### DISCUSSION TITLE
[get_catalog_item]  US/CA:OK GB/IT/DE/FR:OK JP:NG  Why?

### DISCUSSION THREAD

**Discussion by ShakMitz** [saleweaver/python-amazon-sp-api#284] [Q&A]
HI.

get_catalog_item
Although the usage is the same

US / CA is OK,
GB / IT / DE / FR is OK

Nevertheless,
JP becomes NG

why?

error contents
`requests.exceptions.SSLError: HTTPSConnectionPool (host ='sellingpartnerapi-fe.amazon.com', port = 443): Max retries exceeded with url: /catalog/2020-12-01/items/B074XCP65Q?marketplaceIds=A1VC38T7YXB528&includedData=images% 2CproductTypes% 2CsalesRanks% 2Csummaries% 2Cvariations% 2Cattributes% 2Cidentifiers (Caused by SSLError (SSLError (1,'[SSL: SSLV3_ALERT_HANDSHAKE_FAILURE] sslv3 alert handshake failure (_ssl.c: 997)')))`

Python 3.10
python-amazon-sp-api v0.7.4

**Comment by saleweaver** [comment]
The url looks invalid?

**Reply by ShakMitz** [reply]
Thank you for your comment.
This problem has been resolved.

**Reply by saleweaver** [reply]
Can you share how so that it might help others?

**Reply by ShakMitz** [reply]
It was a simple mistake.

When creating an app in Seller Central
"arn: aws: iam :: xxxxxxxxxxxx: role / XXXXXXXXXX" (OK)
"arn: aws: iam :: xxxxxxxxxxxx: user / XXXXXXXXXX" (NG)
It was.

I made a copy and paste mistake at some point.

I'm sorry.

**Reply by saleweaver** [reply]
Ah alright.. 👍  I think that has happened to a lot of people before.
